static/js/src/ua-payment-modal.js

```diff
@@ -762,6 +806,8 @@ function processStripePayment() {
         presentError();
       });
   } else if (currentTransaction.type === "purchase") {
+    checkoutEvent(analyticsFriendlyProducts(), "Confirmed purchase");
```

Comment:
Steps should be integers.
```suggestion
    checkoutEvent(analyticsFriendlyProducts(), 3);
```